components/slider/slider.tsx

```diff
@@ -14,6 +14,7 @@ import SliderDot from './slider-dot'
 import SliderMark from './slider-mark'
 
 interface Props {
+  hideValue?: boolean
```

Comment:
The following default values need to be added to the:
```
hideValue: false,
```

Reply:
Alright i would add default props here: 

https://github.com/geist-org/react/blob/18d08f8accc70a66cc69140846598665b686f0df/components/slider/slider.tsx#L28

and in `docs` update the table for `hideValue` prop

Reply:
Added a Commit should work 

if not let me know more than happy to ammend 

Also, have 2 more PRs to work on this Repo 💙